support-frontend/assets/components/priceCards/priceCards.tsx

```diff
@@ -73,13 +74,24 @@ export function PriceCards({
 	onAmountChange,
 	paymentInterval,
 	otherAmountField,
+	hideChooseYourAmount,
 }: PriceCardsProps): JSX.Element {
-	const otherAmountLabel = amounts.length % 2 ? 'Other' : 'Choose your amount';
+	let currentAmountsLen = hideChooseYourAmount
+		? amounts.length - 1
+		: amounts.length;
+	let otherAmountLabel = currentAmountsLen % 2 ? 'Other' : 'Choose your amount';
+
+	// Check to catch edge case where no amounts supplied and hideChooseYourAmount is true
+	if (hideChooseYourAmount && !amounts.length) {
+		hideChooseYourAmount = false;
```

Comment:
This mutates a prop, which I think is not recommended in react

Reply:
Agreed. Will fix.

Reply:
Fixed